CherkasovaDA/lab_3/main.cpp

```diff
@@ -0,0 +1,260 @@
+#include <iostream>
+
+using namespace std;
+
+struct MyType
+{
+    int a;
+};
+
+class MyVector
+{
+    int size = 0;
+    int capasity = 0;
+    MyType* massiv = 0;
+   
+    MyType NullItem()
+    {
+        MyType ItemNull{ 0 };
+        return ItemNull;
+    }
+    void PrintOneItem(int index)
+    {
+        cout << massiv[index].a;
+    }
+    void IncreasingTheCapasity()
```

Comment:
IncreasingTheCapasity does 2 memory allocations but it's possible to allocate once